pkgs/build-support/rust/import-cargo-lock.nix

```diff
@@ -195,7 +195,7 @@ let
         echo Found crate ${pkg.name} at $crateCargoTOML
         tree=$(dirname $crateCargoTOML)
 
-        cp -prvL "$tree/" $out
+        ${python3}/bin/python -c 'import sys; import shutil; shutil.copytree(sys.argv[1], sys.argv[2], ignore_dangling_symlinks=True)' "$tree" "$out"
```

Comment:
Perhaps you could use `python3Packages.python` instead of `python3` to not have to change to the first line of the file.

Reply:
:+1: Done.